pkg/scheduler/framework/plugins/clusteraffinity/types_test.go

```diff
@@ -627,13 +669,15 @@ func TestClusterRequirementMatches(t *testing.T) {
 		{
 			name: "op <=, not matched",
 			clusterRequirement: &clusterRequirement{
-				PropertySelector: &placementv1beta1.PropertySelector{
-					MatchExpressions: []placementv1beta1.PropertySelectorRequirement{
-						{
-							Name:     propertyprovider.AllocatableCPUCapacityProperty,
-							Operator: placementv1beta1.PropertySelectorLessThanOrEqualTo,
-							Values: []string{
-								"7",
+				ClusterSelectorTerm: placementv1beta1.ClusterSelectorTerm{
```

Comment:
please create a separate test func to add test cases for capacity

eg, TestClusterRequirementMatches_WithPropertyChecker